### ISSUE TITLE
Initialising issues under IE8-11

### ISSUE THREAD

**Issue by halmai** [bassjobsen/Bootstrap-3-Typeahead#149]
Hi, I have problems with the following code under IE. (This is an over-simplified version, just to illustrate the issue.)

    <!DOCTYPE html>
    <html lang="en">
        <head>
            <meta charset="utf-8">
            <title>Bootstrap 3 Type-ahead Demo</title>
            <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
            <script src="bootstrap-3.3.5-dist/js/bootstrap.js"></script>
            <script src="bootstrap3-typeahead.js"></script>
        </head>
        
        <body>
            Type here:
            <input id="typeahead-input" type="text" class="typeahead" data-provide="typeahead" />
            
            <script type="text/javascript">
                jQuery(document).ready(function() {
                    $input = $('#typeahead-input');
                    console.log('starting...', $input);
                    $input.typeahead({
                        source: function (query, process) {
                            console.log(query);
                        }
                    });
                })
            </script>
        </body>
    </html>

If I open this in Firefox (40.0.2) or Chrome (44.0.2403.157 m) then it works fine. When the page is loaded then I click the text area and type "dummy" then the console contains this:

    starting... Object[input#typeahead-input.typeahead property value = "" attribute value = "null"]
    d
    du
    dum
    dumm
    dummy
If after pressing the last y of "dummy" I press Ctrl+F5 then the same happens again. This is cool. 

However, if I do the same in Internet Explorer 11 then the things behave strange. I do the following steps (after turning the console on):
- visit the page
- click the input field
- type "dummy"
- check the console (being happy, "starting...", "d", "du", ..., "dummy" are there)
- press Ctrl+F5 within the input field
- type "dummy" again
- check the console, being SAD, "starting..." is there but no "d", "du", etc.

However #2, if before Ctrl+F5 I click outside of the text box, then everything works fine. 

In a nutshell, after pressing Ctrl+F5 the typeahead field is not initialized properly if I was within the text field but it is if I was outside. This happens in IE8 and IE11 and does not in FF and Chrome.

I'm not sure if it is an IE bug, JQuery bug of typeahead bug.
